### PR TITLE
feat: add error page

### PR DESCRIPTION
**Pull request by Skn0tt**
This PR adds an (undocumented) `/error` endpoint, useful for demonstrating the Netlify-Branded Error Page (which doesn't work yet, but will in the very near future).

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
Thanks @Skn0tt — I'm wondering whether it would actually be more useful to add documentation (when it works) and merge after that?

**Comment by Skn0tt** [comment]
Good question! If this is a feature we want to publically document (sounds like a good idea), then that's a good thing to do probably :)

File: pages/uncaught-exceptions/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export default async () => {
+  throw new Error("💥");
+};
+```
+
+- [Explore the code for this Edge Function](../../netlify/edge-functions/error.ts)
+
+## View this example on the web
+
+- https://edge-functions-examples.netlify.app/example/environment
```
Comment (whitep4nth3r) [inline pages/uncaught-exceptions/README.md:26]: Needs updating to `uncaught-exceptions`

**Comment by whitep4nth3r** [comment]
Looks good to me apart from the one comment!

File: pages/uncaught-exceptions/README.md (hunk 1 above)
Comment (Skn0tt) [reply pages/uncaught-exceptions/README.md:26]:
```suggestion
- https://edge-functions-examples.netlify.app/example/uncaught-exceptions
```

**Comment by whitep4nth3r** [comment]
Just looking at the deploy preview, the screenshot of the error page is odd to me. I know this will be to do with Edge Functions themselves — but it looks odd showing `Error: Error:`

image

**Comment by gitakileus** [comment]
I totally agree!

**Pull request merged**